### ISSUE TITLE
Page title on the my account page moved to bottom

### ISSUE THREAD

**Issue by iGerchak** [magento/magento2#16786]
### Preconditions
1. Magento 2.2.4 or 2.2.5 installed via composer
2. Installed following module: https://github.com/deved-it/magento2-disable-customer-registration#installation

### Steps to reproduce
1. Go to my account page on frontend
2. Look at the page title

### Expected result
1. Page title should be located at the top
![screenshot2](https://user-images.githubusercontent.com/34220204/42697852-ee49ac40-86c4-11e8-936a-3a31fb61dffd.png)

### Actual result
1. Page title shown at the bottom of the page
![screenshot](https://user-images.githubusercontent.com/34220204/42697835-e4371896-86c4-11e8-8f73-75ab9011a401.png)

### Tech details
Root of this issue - in config.php we have Magento_Theme **after** Magento_Customer. In magento 2.2.3 it was Magento_Theme **before** Magento_Customer. If we manually change order in config.php - title moves to top, as expected. Hovewer once we run `php bin/magento setup:upgrade` - order of the modules brakes again.

**Comment by magento-engcom-team** [comment]
Hi @iGerchak. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [x] Summary of the issue
- [x] Information on your environment
- [x] Steps to reproduce
- [x] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me {$VERSION} instance` 

where `{$VERSION}` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (2.2-develop +). 
For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation.

@iGerchak do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [x] yes
- [ ] no

**Comment by lbajsarowicz** [comment]
The problem is not related with Magento itself.
Plugin also does not suggest to break this.
I think it's related with your customizations. Could you provide more information about your custom theme?

<img width="1066" alt="zrzut ekranu 2018-07-13 o 19 46 52" src="https://user-images.githubusercontent.com/1639941/42705990-8ece02aa-86d5-11e8-99dd-5bc2c426c2d8.png">

**Comment by ihor-sviziev** [comment]
@lbajsarowicz look at tech details section. We were able to reproduce it on empty magento installation without any customizations, on luma theme. Module that listed above just have in sequence "customer" module, as result in 2.2.4+ customer module become listed before theme module in config.php, and it fails to execute "move" in https://github.com/magento/magento2/blob/2.2-develop/app/design/frontend/Magento/luma/Magento_Customer/layout/customer_account.xml#L56 because when it tries to move block (that should to be created in theme module) - it wasn't yet created.

PS: Have you executed `php bin/magento setup:upgrade` before going to frontend?

**Comment by engcom-backlog-pb** [comment]
@iGerchak, please refer to the [Community Forums](https://community.magento.com/) or the [Magento Stack Exchange](https://magento.stackexchange.com/) site for advice or general discussion about this issue. The GitHub issue tracker is intended for Magento Core technical issues only.

**Comment by ghost** [comment]
@iGerchak, thank you for your report.
We've acknowledged the issue and added to our backlog.

**Comment by chirag-wagento** [comment]
@magento-engcom-team give me test instance

**Comment by ihor-sviziev** [comment]
@chirag-wagento AFAIK it works only in PRs

**Comment by osrecio** [comment]
@ihor-sviziev yes, now it's work the command in issues 😉 :

I reproduced the error too

```
@magento-engcom-team give me {$VERSION} instance

where {$VERSION} is version tags (starting from 2.2.0+) or develop branches (2.2-develop +).
For more details, please, review the Magento Contributor Assistant documentation.
```

**Comment by southerncomputer** [comment]
happens since magento 2.1, order of modules causes this plus my orders to flip address below items in my account

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if you want to validate it one more time, please, go though the following instruction:
- [x] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [x] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by mitalikundale** [comment]
Please give me 2.1 and 2.2 develop instance to reproduce the issue.

**Comment by magento-engcom-team** [comment]
Hi @iGerchak, @mitalikundale. 

Thank you for your report and collaboration!

The related internal Jira ticket `MAGETWO-93518` was closed as `non-reproducible` in `2.2.x`.
It means that Magento team either unable to reproduce this issue using provided _Steps to Reproduce_ from the **_Description section_** on clean or the issue has been already fixed in the scope of other tasks.

But if you still run into this problem please update or provide additional information/steps/preconditions in the **_Description section_** and reopen this issue.

**Comment by edenduong** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @edenduong. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @edenduong, here is your [Magento instance](https://i-16786-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-16786-2-3-develop.instances.magento-community.engineering/admin](https://i-16786-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by chandu-pinpoint** [comment]
@magento-engcom-team It looks fine on 2.3.x

**Comment by sdzhepa** [comment]
Closed as cannot reproduce on the latest `2.4-develop` code.
But if you still faced the issue please create a new report for 2.4 with detailed steps
